src/main/java/org/synyx/urlaubsverwaltung/absence/web/AbsenceOverviewViewController.java

```diff
@@ -105,14 +105,36 @@ public String absenceOverview(
 
         final Person signedInUser = personService.getSignedInUser();
 
-        final List<Department> departmentsOfUser = departmentService.getAllowedDepartmentsOfPerson(signedInUser);
-        model.addAttribute("departments", departmentsOfUser);
-
-        final String fallbackDepartment = departmentsOfUser.isEmpty() ? "" : departmentsOfUser.get(0).getName();
-        final List<String> preparedSelectedDepartments = rawSelectedDepartments.stream().filter(StringUtils::hasText).collect(toList());
-        final List<String> selectedDepartmentNames = preparedSelectedDepartments.isEmpty() ?
-            List.of(fallbackDepartment) : preparedSelectedDepartments;
-        model.addAttribute("selectedDepartments", selectedDepartmentNames);
+        final List<Person> overviewPersons;
+        final List<Department> allDepartments = departmentService.getAllDepartments();
```

Comment:
Könnte ein größerer DB-Call sein mit allen Departments und damit verbunden Persons usw, das Ergebnis wird in 90% der Fälle (außer Boss und Office) verworfen/nicht verwendet.

Reply:
gut gesehen, hier wollte ich noch eine Abfrage auf `count` machen.